frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -897,7 +897,7 @@ function ReaderDictionary:stardictLookup(word, dict_names, fuzzy_search, boxes,
         end
 
     -- Event for plugin to catch lookup with book title
-    self.ui:handleEvent(Event:new("WordLookedUp", word, book_title))
+    self.ui:handleEvent(Event:new("WordLookedUp", word, book_title, pos))
```

Comment:
(Some too munch indentation in the section just above (that you may have left to limit the diff in the previous PR, but well, it jumped at me :)